partcad/src/partcad/test/cam.py

```
@@ -15,6 +15,7 @@
 from ..part_config import PartConfiguration
 from ..assembly import Assembly
 from ..assembly_config import AssemblyConfiguration
+from ..provider_data_cart import ProviderCart, ProviderCartItem
```

Ruff (0.8.2)

`..provider_data_cart.ProviderCart` imported but unused

Remove unused import: `..provider_data_cart.ProviderCart`

(F401)